tools/StaticAnalysis/StaticAnalysis.Netcore.csproj

```diff
@@ -22,7 +22,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```

Comment:
This csproj references `Tools.Common.Netcore.csproj`, `Tools.Common.Netcore.csproj` references `Newtonsoft.Json`, can we remove this line?

Reply:
Removed.